### ISSUE TITLE
resolve duplicates before listing errors

### ISSUE THREAD

**Issue by teixeirak** [SCBI-ForestGEO/SCBImortality#38]
@ValentineHerr , we seem to have cases where there are duplicates-- one incorrect and one correct-- where both show up as required field fixes. Examples I discovered today included 80694 and 80770 in quadrat 816.

**Comment by ValentineHerr** [comment]
should be done now.

**Comment by teixeirak** [comment]
@ValentineHerr , I'm wondering if the correct duplicate is being selected. In the example below ( quad 815-816), I checked all those trees on Wednesday and think I resolved all the issues (certainly didn't miss so many!). 

<img width="809" alt="image" src="https://user-images.githubusercontent.com/6355854/125973698-3300b270-7dd6-46db-b03a-49bf21e9d90c.png">

There were duplicates for all these trees (plots had accidentally been censused twice), so this could be an issue on the FastField end...  Or perhaps the good version of the record got thrown out by your code? I'm not sure what's going on. This may require going back to check the previous files.
